lib/Adapter/WorseReflection/Refactor/WorseExtractMethod.php

```diff
@@ -205,8 +205,17 @@ private function addParametersAndGetArgs(array $freeVariables, $methodBuilder, S
             $variableType = $freeVariable->symbolContext()->type();
 
             if ($variableType->isDefined()) {
-                $parameterBuilder->type($variableType->short());
+                // TODO Create a utils class or something to get the shortest type by taking
+                // into account the import table, to handle aliases and partial namespaces
+                $shortTypeAsString = ($variableType->isNullable() ? '?' : '') . $variableType->short();
```

Comment:
can we remove the TODO. This is to do with the qualified names topic :)

Reply:
Will do